docs/Writing-a-module.md

```diff
@@ -13,12 +13,11 @@ Every module surfaces on the admin settings page of the performance plugin, wher
 
 ## Module requirements
 
-* The production code for a module must all be located in a directory `/modules/{module-slug}` where `{module-slug}` is the module's slug.
-* The entry point file must be called `load.php` and per the above be located at `/modules/{module-slug}/load.php`.
+* The production code for a module must all be located in a directory `/modules/{focus}/{module-slug}` where `{module-slug}` is the module's slug and `{focus}` is the focus area: an identifier of a single focus (e.g. `images`). This should correspond to a section on the performance plugin's settings page. [See the `perflab_get_focus_areas()` function for the currently available focus areas.](../admin/load.php#L161)
+* The entry point file must be called `load.php` and per the above be located at `/modules/{focus}/{module-slug}/load.php`.
```

Comment:
This change has to also be made in a few other places in this file, e.g. in the tests instructions and also in the location for the demo file. Please double-check to ensure they are all updated.

Reply:
👍🏼  I wasn't sure we wanted to change test structure as well; will update